README.rst

```diff
@@ -12,20 +12,26 @@ Extracts information from life-science websites and texts, generating phenopacke
 Setup
 -----
 
+To get the project's source code, clone the github repository:
+
+::
+
+  $ git clone https://github.com/monarch-initiative/phenopacket-scraper-core.git
+
 First, you need to create a virtual environment and activate it.
 
 ::
 
-  $ pip install virtualenv
+  $ sudo pip install virtualenv
```

Comment:
Is the `sudo` really necessary? I did not need `sudo` when I installed virtualenv. That might be because my Python installation is via HomeBrew, which uses non-system permissions and ownerships for the files. You might want to add a note to the README saying that `sudo` may not be required.